Tests/StackClusterTests.swift

```diff
@@ -171,10 +173,236 @@ class StackClusterTests: QuickSpec {
                 }
             }
             
+            context("when scrolling horizontally") {
+                let size = CGSize(width: 200, height: 300)
+                let children = [
+                    labelComponent(title: "1", color: .purple, labelSize: size),
+                    labelComponent(title: "2", color: .red, labelSize: size),
+                    labelComponent(title: "3", color: .yellow, labelSize: size),
+                    labelComponent(title: "4", color: .green, labelSize: size),
+                    labelComponent(title: "5", color: .blue, labelSize: size)
+                ]
+                
+                let meta = ClusterLayout.StackConfig(axis: .horizontal)
+                let vc = stack(with: children, meta: meta)
+                vc?.loadViewIfNeeded()
+                vc?.view.layoutIfNeeded()
+                
+                let purpleLeftmostTargetVC = vc?.childViewControllers[0]
+                
+                it("a leftmost target gets left aligned, if it could not get centered correctly") {
+                    vc?.scrollToViewController(target: purpleLeftmostTargetVC!, position: .center, animated: false)
+                    expect(vc).to(haveValidSnapshot())
+                }
+                
+                it("a leftmost target gets left aligned, if it could not get right aligned correctly") {
```

Comment:
me too, please use something like:
context when ....... (if needed)
   it should.....